front/src/views/NotFoundView/NotFoundView.tsx

```diff
@@ -1,9 +1,20 @@
+import { useAuth0 } from "@auth0/auth0-react";
+import { useErrorHandling } from "hooks/useErrorHandling";
 import { Navigate, useLocation } from "react-router-dom";
 
 function NotFoundView() {
+  const { user } = useAuth0();
   const location = useLocation();
-  // If the requested route was not found redirect to /
-  return <Navigate to="/" replace state={{ origin: location.pathname }} />;
+  const { triggerError } = useErrorHandling();
+
+  if (!user || !user["https://www.boxtribute.com/base_ids"]) {
+    triggerError({ message: "This requested page does not exist." });
+    return <div>Not Found</div>;
+  }
+
+  // If the requested route was not found redirect to statviz
+  const baseId = user["https://www.boxtribute.com/base_ids"][0];
```

Comment:
I added a commit for this, but in most cases it will not make a difference. If you do a Browser reload (only way to test this atm through the UI all states are flushed and it does not remember the selected base from before.

@fhenrich33 will jotai change this?